pkg/ivd/ivd_protected_entity_type_manager_test.go

```diff
@@ -99,15 +100,29 @@ func getVcUrlFromConfig(config *rest.Config) (*url.URL, bool, error) {
 	return vcUrl, insecure, nil
 }
 
-func verifyMdIsRestoredAsExpected(md metadata) bool {
-	reservedLabels := []string {
-		"cns.containerCluster.clusterFlavor",
-		"cns.containerCluster.clusterId",
-		"cns.containerCluster.clusterType",
-		"cns.containerCluster.vSphereUser",
-		"cns.k8s.pv.name",
-		"cns.tag",
-		"cns.version",
+func verifyMdIsRestoredAsExpected(md metadata, version string, logger logrus.FieldLogger) bool {
+	var reservedLabels []string
+	if strings.Contains(version, "6.7U3") {
+		reservedLabels = []string {
+			"cns.clusterID",
+			"cns.clusterType",
+			"cns.vSphereUser",
+			"cns.k8s.pvName",
+			"cns.tag",
+		}
+	} else if strings.Contains(version, "7.0") {
+		reservedLabels = []string {
+			"cns.containerCluster.clusterFlavor",
+			"cns.containerCluster.clusterId",
+			"cns.containerCluster.clusterType",
+			"cns.containerCluster.vSphereUser",
+			"cns.k8s.pv.name",
+			"cns.tag",
+			"cns.version",
+		}
+	} else {
+		logger.Debug("Newer VC version than what we expect. Skip the verification.")
```

Comment:
Let's print the VC version we're seeing

Reply:
I printed the VC version and URL at the beginning of the test case.